src/debugger/nodeDebugWrapper.ts

```diff
@@ -195,9 +199,11 @@ export class NodeDebugWrapper {
                     const sourceMaps = debugSession._sourceMaps;
                     if (sourceMaps) {
                         // Flush any cached source maps
-                        sourceMaps._allSourceMaps = {};
-                        sourceMaps._generatedToSourceMaps = {};
-                        sourceMaps._sourceToGeneratedMaps = {};
+                        // Rather than cleaning internal caches we recreate
+                        // SourceMaps to add downloaded bundle map to cache
+                        const bundlePattern = path.join(sourcesDir, "index.*.bundle");

```

Comment:
We don't guarantee that the source file will be `index.[ios|android].bundle`, we let the native code define it and make whatever request it wants. I'd suggest just using `*.bundle` as the glob.